ts/features/itwallet/machine/eid/context.ts

```diff
@@ -8,9 +8,17 @@ import { IssuanceFailure } from "./failure";
 type WithAbortController<T> = T & { abortController?: AbortController };
 
 export type IdentificationContext =
-  | WithAbortController<{ mode: "cieId" }>
-  | WithAbortController<{ mode: "ciePin"; pin: string }>
-  | WithAbortController<{ mode: "spid"; idpId: string }>;
+  | WithAbortController<{ mode: "cieId"; level: "L2" }>
```

Comment:
Since we are here, I think we no longer use the `AbortController`, so we can safely remove every reference to it.

Reply:
Removed from `IdentificationContext`, thanks @gispada!

Reply:
I cleaned up everything related to it, thanks @gispada!